File: hw/bonilla/u3/hw35StrategyPersistence/StrategyJS/model/parking.js
export default class Parking {
    constructor(id, plate, vehicleType, entryTime, exitTime, fee) {
        this.id = id;
        this.plate = plate;
        this.vehicleType = vehicleType;
        this.entryTime = entryTime ? new Date(entryTime) : null;
        this.exitTime = exitTime ? new Date(exitTime) : null;
        this.fee = fee || 0;
    }

    toString() {
        return `${this.plate} (${this.vehicleType})`;
    }
}

File: hw/bonilla/u3/hw35StrategyPersistence/StrategyPy/model/parking.py
from datetime import datetime

class Parking:
    def __init__(self, id, plate, vehicle_type, entry_time, exit_time=None, fee=0.0):
        self.id = id
        self.plate = plate
        self.vehicle_type = vehicle_type
        self.entry_time = entry_time
        self.exit_time = exit_time
        self.fee = fee

    def __str__(self):
        return f"{self.plate} ({self.vehicle_type})"